{{cookiecutter.project_slug}}/src/{{cookiecutter.project_module}}/errorhandlers.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2018, Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Error handlers and custom exceptions.
+
+http://flask.pocoo.org/docs/1.0/errorhandling/
+http://flask.pocoo.org/docs/1.0/patterns/apierrors/
+"""
+
+import logging
+import sys
+
+from flask import jsonify
+
+from werkzeug.exceptions import HTTPException
+from werkzeug.routing import RoutingException
+
+
+logger = logging.getLogger(__name__)
+
+
+def init_app(app):
```

Comment:
What's the reason for using an `init_app` function rather than simply importing `app`?

Reply:
This is to allow the app module a "reason" to import the module. Otherwise, you'd have to do something like `from . import errorhandlers  # noqa` in `app.init_app`, and then importing the module would have the side effect of registering the handlers through the decorators. That appears a bit odd to me and most linters will warn about unused import.

Besides, as you know this is a fairly common idiom for flask extensions, and I think it makes sense to treat internal modules the same.